test/types/range/userAPI/partialInit.chpl

```diff
@@ -4,10 +4,10 @@ proc main() {
   // expression
   type SR = range(strides=strideKind.any, ?);
 
-  var A : SR = 1..10;
+  var A : SR(?) = 1..10;
   writeln(A.type:string, ": ", A);
 
-  var B : SR = 1:uint .. 10:uint;
+  var B : SR(?) = 1:uint .. 10:uint;
```

Comment:
Is it clearer to you than to me (which is to say "not at all"):  What makes cases like this get caught by this PR and not be your previous work that tried to alert to variables of generic type?

Reply:
It's because of the change in `emitTypeAliasInit` in normalize.cpp. SR was defined as `type SR = range(strides=strideKind.any, ?);` and the logic in normalize was considering `B` marked with `(?)` because of the `?` in the type declaration for `SR`.